ux.symfony.com/templates/ux_packages/map.html.twig

```diff
@@ -3,15 +3,18 @@
 {% set package = 'map'|ux_package %}
 
 {% block package_header %}
-    <twig:Hero
-        eyebrows="Assert('Mapper')"
-        title="UX Map"
-        url="{{ url('app_map') }}"
-    >
-        Embed <code>interactive maps</code> in your Symfony application in a breeze!
-        <br />
-        Decouple your code from your map provider: <code>Google Maps</code>, <code>LeaftLet</code>.
-    </twig:Hero>
+    {% component PackageHeader with {
+        package: 'map',
+        eyebrowText: 'Seamless Maps Integration',
+    } %}
+        {% block title_header %}
+            Embed <em>interactive maps</em> in a breeze!
+        {% endblock %}
+
+        {% block sub_content %}
+            Decouple your code from your map provider: <em>Google Maps</em>, <em>LeaftLet</em>.
+        {% endblock %}
+    {% endcomponent %}
```

Comment:
Why not use the HTML syntax?

Reply:
It's not used elsewhere for now. I used it in the first version of this page with the new component but here it feels more coherent.

But all this will really soon change so i don't mind :)

Reply:
I've just go for the fastest (all other packages use this syntax), hoping this PR could quickly be merged and deployed :)